Core/src/org/sleuthkit/autopsy/keywordsearchservice/KeywordSearchService.java

```diff
@@ -97,4 +97,13 @@ default public void close() throws IOException {
          */
     }    
 
+    /**
+     * Deletes the keyword search text for a specific data source.
+     *
+     * @param dataSourceId The data source id to be deleted from Solr.
```

Comment:
Technically, this interface knows nothing about Solr.

Reply:
Changed comment to remove Solr from it.